### PR TITLE
i18n(ko-KR): update `sidebar.mdx`

### PR DESCRIPTION
**Pull request by jsparkdev**
#### Description

- update `sidebar.mdx`
- #2285

### CODE REVIEW

File: docs/src/content/docs/ko/guides/sidebar.mdx [hunk 1]
```diff
@@ -518,6 +518,52 @@ starlight({
...
+### 배지 국제화
+
+[배지](#배지)의 경우 `text` 속성은 문자열이며, 다국어 사이트의 경우 각 로케일별 값이 포함된 객체일 수 있습니다.
+객체 양식을 사용하는 경우 키는 [BCP-47](https://www.w3.org/International/questions/qa-choosing-language-tags) 태그 (예: `en`, `ar` 또는 `zh-CN`)여야 합니다:
+
+```js {11-16}
+starlight({
+	sidebar: [
+		{
+			label: 'Constellations',
```
Comment (HiDeoo) [inline docs/src/content/docs/ko/guides/sidebar.mdx:530]:
```suggestion
			label: '별자리',
```

I guess the label could be translated like the previous i18n example? What do you think?

File: docs/src/content/docs/ko/guides/sidebar.mdx [hunk 2]
```diff
@@ -518,6 +518,52 @@ starlight({
...
+			translations: {
+				'pt-BR': 'Constelações',
+			},
+			items: [
+				{
+					slug: 'constellations/andromeda',
+					badge: {
+						text: {
+							en: 'New',
```
Comment (HiDeoo) [inline docs/src/content/docs/ko/guides/sidebar.mdx:539]: If we translate the label, I guess it would make sense to change to `ko: '…'`?

**Review by HiDeoo — COMMENTED** [review]
Thanks for the translation :raised_hands:

As a non-native speaker, I reviewed this PR using the [Docs Translation Review Checklist](https://hackmd.io/1KYnrpUQR0SAGTzHIUoUDg?view). Left a few comments regarding translating the group label to match the other i18n example but other than that, this looks perfect to me.

File: docs/src/content/docs/ko/guides/sidebar.mdx (hunk 1 above)
Comment (jsparkdev) [reply docs/src/content/docs/ko/guides/sidebar.mdx:530]: Thank you!

File: docs/src/content/docs/ko/guides/sidebar.mdx (hunk 2 above)
Comment (jsparkdev) [reply docs/src/content/docs/ko/guides/sidebar.mdx]: Thank you. I've fixed it.

**Review by HiDeoo — APPROVED** [review]
Thanks for the fast update, looks perfect to me :raised_hands:

**Pull request merged**